test/helpers/components/dynakube/dynakube.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const (
 	defaultName      = "dynakube"
-	defaultNamespace = "dynatrace"
+	DefaultNamespace = "dynatrace"
```

Comment:
```suggestion
	defaultNamespace = "dynatrace"
```

should not export it, try to rely on the dynakube instance's `Namespace`